addons/web/static/src/js/views/form_widgets.js

```diff
@@ -888,6 +888,23 @@ var FieldSelection = common.AbstractField.extend(common.ReinitializeFieldMixin,
     }
 });
 
+var TimeZone = FieldSelection.extend({
+    initialize_content: function() {
+        var self = this;
```

Comment:
May be better to put whole stuff in render_value, render_value will be called on value change